cluster/gce/container-linux/configure-helper.sh

```diff
@@ -266,12 +266,12 @@ kind: Config
 users:
 - name: kubelet
   user:
-    client-certificate-data: ${KUBELET_CERT}
-    client-key-data: ${KUBELET_KEY}
+    token: ${KUBELET_TOKEN}
 clusters:
 - name: local
   cluster:
     certificate-authority-data: ${KUBELET_CA_CERT}
+    server: https://${KUBERNETES_MASTER_NAME}:443
```

Comment:
note that we don't always pass the --apiservers flag to the kubelet -- it depends on whether the kubelet is on a node or the master and if the environment variable for the master is set. So if we want to remove that flag from our scripts, we'll need to add logic here (and in the gci scripts) to *conditionally* add this line, preserving the same logic that we have today.